elements/rh-secondary-nav/rh-secondary-nav.ts

```diff
@@ -106,6 +108,12 @@ export class RhSecondaryNav extends LitElement {
   @colorContextProvider()
   @property({ reflect: true, attribute: 'color-palette' }) colorPalette: NavPalette = 'lighter';
 
+  /**
+   * If the host color-palette attr is set to lighter, the cta color context should be lighest
+   * Otherwise use the the same as value as the host color-palette attr as default set value.
+   */
+  @state() private _ctaColorPalette = this.colorPalette === 'lighter' ? 'lighest' : this.colorPalette;
```

Comment:
please add a little more info here about the problem, so that we can be confident in removing the workaround down the line if we're able.
```suggestion
  // NB: yadda yadda mobile dropdown context, and that this can be removed if design adds a dark bg
  @state() private _ctaColorPalette = this.colorPalette === 'lighter' ? 'lighest' : this.colorPalette;
```